src/components/ImageViewer.ts

```diff
@@ -62,11 +63,9 @@ class ImageViewer extends Component<ImageViewerProps, ImageViewerState> {
     }
 
     private toggleLightBox() {
-        if (this.props.onClickOption === "openFullScreen") {
-            this.setState({
-                isOpen: !this.state.isOpen
-            });
-        }
+        this.setState({
+            isOpen: !this.state.isOpen
+        });
     }
 
     private getStyle(value: string | number, type: string): number | string {
```

Comment:
if statements could be `else if` please add a comment that `When type is auto default browser styles applies`